doc/source/using-ray-with-tensorflow.rst

```diff
@@ -320,3 +320,38 @@ For reference, the full code is below:
     # and 0.3 used in generate_fake_x_y_data.
     if iteration % 20 == 0:
       print("Iteration {}: weights are {}".format(iteration, weights))
+
+.. autoclass:: ray.experimental.TensorFlowVariables
+   :members:
+
+Troubleshooting
+---------------
+
+Note that ``TensorFlowVariables`` uses variable names to determine what variables get set when calling ``set_weights``.
+One common issue when sharing weights between networks is that the setter and getter networks are both defined
+in same Tensorflow graph. This occurs as Tensorflow appends an underscore and integer to the names of variables
+whose name is already taken in the graph. For example, if I had a class definiton ``Network`` with a ``TensorFlowVariables`` 
+instance inside and ran the following code:
+
+.. code-block:: python
+
+  a = Network()
+  b = Network()
+  b.set_weights(a.get_weights())
+
+the code would fail. If I instead defined each network in its own Tensorflow graph, then it would work:
+
+.. code-block:: python
+
+  with tf.Graph().as_default():
+    a = Network()
+  with tf.Graph().as_default():
+    b = Network()
+  b.set_weights(a.get_weights())
+
+This issue does not occur between actors that contain a network, as each actor is in its own process, and thus is in
+its own graph. 
+
+Another thing to keep in mind when using the class is that creating a ``TensorFlowVariables`` adds new operations to the graph,
```

Comment:
Another issue to keep in mind is that ``TensorFlowVariables`` needs to add new operations to the graph. If you close the graph...